skietbaan_fe/src/components/ViewMembers.js

```diff
@@ -33,11 +36,28 @@ class ViewMembers extends Component {
             }).then(data => this.setState({
                 array: data
             }))
-            .catch(function() { });
+            .catch(function () { });
+    }
+
+
+    GetFilteredMembers() {
```

Comment:
This function should look something like this 
```
fetch(BASE_URL + "/api/Features/SearchMember", {
	method: 'Get',
	headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' },
}).then(function (response) {
	return response.json();
}).then(function (data) {
	return data;
}).then(data => this.setState({
	array: data
})).catch(function () { });
```

Reply:
Also, why catch if you aren't doing anything?

Reply:
okay removed all the catches